src/cct3.cc

```diff
@@ -411,6 +411,7 @@ SharedWavefunction psi4_cct3(SharedWavefunction ref_wfn, Options& options)
         eneprim = "CC(t;3)";
     }
 
+    ref_wfn->set_module("cct3");
```

Comment:
This doesn't seem to be a member function. Is this something from a newer version of Psi4?

Reply:
ah, yes, just merged yesterday: https://github.com/psi4/psi4/pull/1935

you can comment it out for now. I'll move the setting to py-side and try/except it to be more resilient to broader psi versions.